src/reanimated2/component/FlatList.tsx

```diff
@@ -12,31 +12,30 @@ import { LayoutAnimationConfig } from './LayoutAnimationConfig';
 
 const AnimatedFlatList = createAnimatedComponent(FlatList as any) as any;
 
-interface AnimatedFlatListProps {
+interface CellRendererComponentProps {
   onLayout: (event: LayoutChangeEvent) => void;
   // implicit `children` prop has been removed in @types/react^18.0.0
   children: React.ReactNode;
-  inverted?: boolean;
-  horizontal?: boolean;
+  style?: StyleProps;
 }
 
-const createCellRenderer = (
-  itemLayoutAnimation?: ILayoutAnimationBuilder,
-  cellStyle?: StyleProps
+const createCellRendererComponent = (
+  itemLayoutAnimation?: ILayoutAnimationBuilder
 ) => {
-  const cellRenderer = (props: AnimatedFlatListProps) => {
+  const CellRendererComponent = (props: CellRendererComponentProps) => {
+    console.log(itemLayoutAnimation);
```

Comment:
I think we want to remove this console log 😅

Reply:
😭